cluster-operator/src/test/java/io/strimzi/operator/cluster/operator/assembly/KafkaAssemblyOperatorTest.java

```diff
@@ -429,10 +429,10 @@ private void deleteCluster(TestContext context, KafkaAssembly clusterCm, List<Se
 
             /*Set<String> metricsNames = new HashSet<>();
```

Comment:
If this is commented out anyway why are we still maintaining it?

Reply:
@tombentley Well, I guess IntelliJ refactoring does this - it probably expects us to uncomment this one day.  We can either merge it or I can delete all the unused sections although I'm not sure about their purpose. There is a ton of them. As you want.

Reply:
We should delete them, I guess. They were once testing usingful stuff, but as the CO got refactored some of the assertions became untestable (hence the comments). In the meantime I wrote the MockTest, which is beeter able to test of of these things.